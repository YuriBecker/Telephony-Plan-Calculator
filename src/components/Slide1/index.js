import React from 'react';
import { View, Image, StyleSheet } from 'react-native';
import { Text } from '@ui-kitten/components';
import Images from '../../constants/images';
import colors from '../../constants/colors';

const styles = StyleSheet.create({
  wrapper: {},
  slide: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: colors.green[1],
  },
  text: {
    color: colors.primary,
    fontWeight: 'bold',
  },
});

const index = () => {
  return (
    <View style={styles.slide} level={20}>
      <Text category="h1" style={styles.text}>
        FaleMais
      </Text>
      <Text category="h5" style={styles.text}>
        Planos de Telefonia
      </Text>

      <Image
        style={{
          width: 150,
          height: 50,
          resizeMode: 'stretch',
          position: 'absolute',
          bottom: 50,
        }}
        source={Images.vizerLogo}
      />
    </View>
  );
};

export default index;
