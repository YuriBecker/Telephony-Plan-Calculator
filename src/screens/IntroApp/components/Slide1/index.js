import React from 'react';
import { View, Image, StyleSheet } from 'react-native';
import { Text } from '@ui-kitten/components';
import Images from '../../../../constants/images';
import colors from '../../../../styles/colors';
import FaleMaisLogo from '../../../../components/FaleMaisLogo';

const styles = StyleSheet.create({
  wrapper: {},
  slide: {
    flex: 1,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: colors.black[1],
  },
  text: {
    color: colors.primary,
    fontWeight: 'bold',
  },
});

const index = () => {
  return (
    <View style={styles.slide} level={20}>
      <FaleMaisLogo iconWidth={38} iconHeight={38} />
      <Text category="p2" style={styles.text}>
        Planos de Telefonia
      </Text>

      <Image
        style={{
          width: 150,
          height: 50,
          resizeMode: 'stretch',
          position: 'absolute',
          bottom: 50,
        }}
        source={Images.vizerLogo}
      />
    </View>
  );
};

export default index;
