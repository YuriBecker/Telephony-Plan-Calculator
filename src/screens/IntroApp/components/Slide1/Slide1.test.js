import React from 'react';
import { shallow } from 'enzyme';

import Slide1 from './index';

describe('Testing Slide1 component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Slide1 />);
  });

  it('should render correctly', () => expect(wrapper).toMatchSnapshot());

  it('should render the Vizir Image', () => {
    expect(
      // wrapper.findWhere(node => node.prop('testID') === 'title')
      wrapper.find('Image')
    ).toExist();
  });
});
