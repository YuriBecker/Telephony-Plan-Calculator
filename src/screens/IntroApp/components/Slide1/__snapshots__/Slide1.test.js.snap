// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Slide1 component should render correctly 1`] = `
<View
  level={20}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#212121",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <index
    iconHeight={38}
    iconWidth={38}
    titleCategory="h1"
  />
  <TextComponent
    category="p2"
    style={
      Object {
        "color": "#1db954",
        "fontWeight": "bold",
      }
    }
  >
    Planos de Telefonia
  </TextComponent>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/vizerLogo.png",
      }
    }
    style={
      Object {
        "bottom": 50,
        "height": 50,
        "position": "absolute",
        "resizeMode": "stretch",
        "width": 150,
      }
    }
  />
</View>
`;
